app/src/main/java/com/codesoom/assignment/task/controller/TaskController.java

```diff
@@ -0,0 +1,51 @@
+package com.codesoom.assignment.task.controller;
+
+import com.codesoom.assignment.task.model.domain.Task;
+import com.codesoom.assignment.task.model.request.TaskRequest;
+import com.codesoom.assignment.task.service.TaskService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Set;
+
+import static org.springframework.http.ResponseEntity.ok;
+
+@RestController
+@RequestMapping("/tasks")
+public class TaskController {
+
+    private final TaskService taskService;
+
+    public TaskController(TaskService taskService) {
+        this.taskService = taskService;
+    }
+
+    @GetMapping("/{id}")
+    public ResponseEntity<Task> findById(@PathVariable final long id) {
+        return ok(taskService.findById(id));
+    }
```

Comment:
Spring WebMVC에는 응답에 대한 타입을 바꿔주므로 응답할 때 ResponseEntity객체로 만들어서 반환하지 않아도 됩니다.

```suggestion
    @ResponseStatus(HttpStatus.OK)
    @GetMapping("/{id}")
    public Task findById(@PathVariable final long id) {
        return taskService.findById(id);
    }
```